sycl/test-e2e/lit.cfg.py

```diff
@@ -890,6 +890,13 @@ def open_check_file(file_name):
         ("%clang", " " + config.dpcpp_compiler + " " + config.c_flags)
     )
 
+lit_config.note(
```

Comment:
How about we print all features only if `--verbose` / `-v` is specified?

Reply:
I don't think it would be useful that much. This PR only makes sense if we enable it by default in all our CI jobs, meaning the noise will be there.

Reply:
> I don't think it would be useful that much. This PR only makes sense if we enable it by default in all our CI jobs, meaning the noise will be there.

IIUC, we already pass `-v` to `llvm-lit` in our CI jobs: https://github.com/intel/llvm/blob/sycl/devops/actions/run-tests/e2e/action.yml#L62.
I don't mind printing features in CI jobs as they would be useful for debugging (it's not that noisy :)). My preference would be to not print features unnecessarily when we invoke `llvm-lit` locally. We already print device aspects unconditionally and `llvm-lit`'s output (even without `--verbose`) can be big when we have multiple devices in the environment.